### PR TITLE
added radius prop to VictoryPie per issue #910

### PR DESCRIPTION
**Pull request by BotScutters**
Added support for radius prop to Victory-Pie. If radius prop is not present, radius is calculated from width, height, and padding as before.
-Added radius prop to propTypes in victory-pie.js
-Added check to getRadius in helper-methods.js

Closes https://github.com/FormidableLabs/victory/issues/910

### CODE REVIEW

**Comment by boygirl** [comment]
Please add an `origin` prop that will be an object like `{ x: number, y: number }`. When this prop is provided it will be used rather than calculated from width / height / padding.

**Comment by BotScutters** [comment]
Added support for origin prop to Victory-Pie. If origin prop is not present, origin is calculated from width, height, and padding as before.
-Added origin prop to propTypes in victory-pie.js as an array containing x and y, with x and y able to be either numbers or functions. Should they actually just be numbers and not allow functions?
-Added check in helper-methods.js for whether or not origin prop is provided

Closes FormidableLabs/victory#910

**Comment by boygirl** [comment]
The values for origin.x and origin.y should be restricted to non-negative numbers. Functions don’t make sense here.

File: src/components/victory-pie.js [hunk 1]
```diff
@@ -102,6 +102,16 @@ class VictoryPie extends React.Component {
     labelRadius: PropTypes.oneOfType([ CustomPropTypes.nonNegative, PropTypes.func ]),
     labels: PropTypes.oneOfType([ PropTypes.func, PropTypes.array ]),
     name: PropTypes.string,
+    origin: PropTypes.arrayOf(PropTypes.shape({
```
Comment (boygirl) [inline src/components/victory-pie.js]: This should not be an array of objects. Just a single one.

File: src/components/victory-pie.js [hunk 2]
```diff
@@ -102,6 +102,16 @@ class VictoryPie extends React.Component {
...
+      x: PropTypes.oneOfType([
+        PropTypes.func,
+        CustomPropTypes.allOfType([CustomPropTypes.integer, CustomPropTypes.nonNegative])
+      ]),
+      y: PropTypes.oneOfType([
+        PropTypes.func,
```
Comment (boygirl) [inline src/components/victory-pie.js]: These can just be CustomPropTypes.nonNegative

**Pull request merged**